cmd/kubeadm/app/constants/constants.go

```diff
@@ -306,7 +306,7 @@ const (
 	KubeletHealthzPort = 10248
 
 	// MinExternalEtcdVersion indicates minimum external etcd version which kubeadm supports
-	MinExternalEtcdVersion = "3.4.3-0"
+	MinExternalEtcdVersion = "3.4.13-4"
```

Comment:
`3.4.13-4` is not promoted yet. I opened https://github.com/kubernetes/k8s.io/pull/5440 to promote it.

Reply:
ok, we can hold this PR until it's promoted.